### ISSUE TITLE
Assignment 1 Moritz Engelhardt

### ISSUE THREAD

**Issue by moengels** [EstherPlomp/TNW-RDM-101#43]
### Introduction
Hi, I'm Moritz Engelhardt, a 2nd year PhD student at BN from Germany. 

### Describe your research in 2-3 sentences to someone that is not from your field (please avoid abbreviations)

My research entails on the one hand developing new light microscopy methods and on the other studying protein aggregation dynamics in Huntington's disease. My approach centres around the aspect that aggregation in microscopy up to now has been studied with labels that affect the aggregation process, wherefore I will explore label-free methods and structure-inert labelling methods. 

### My research entails the following aspects: 

| Research Aspect | Answer |
| ------------- | ------------- |
| Use/collect personal data (health data, interviews, surveys)  | No  |
| Use/collect experimental data (lab experiments, measurements with instruments)  | Yes  |
| Collaborate with industry  | No  |
| Write/develop software as the main output of the project  | Yes |
| Use code (as in programming) for data analysis | Yes  |
| Work with large data (images, simulation models) | Yes  |
| Other:  | N/A  |

### Reflections on the importance of RDM videos

I see the reframing towards the selfish aspect of RDM, however I feel that everybody that acknowledges the relevance is already sitting in this course. Overall I think it is the time and resource aspect that keeps people from doing it, rather than maliciousness. 
I myself had to erase parts of our measurement PC and set it up again so keeping the custom scripts on github was a saviour. 

### Checklist assignments 

- [x] Assignment 1: creating a GitHub issue (before **Class 1**)
- [ ] Respond to the GitHub issue on '[data challenge](https://github.com/EstherPlomp/TNW-RDM-101/discussions/33)'
- [ ] [Assignment 2](https://estherplomp.github.io/TNW-RDM-101/06-Assignment-Data-Flow-Map-1.html): Data Flow Map 1 (share a link in this issue before **7 March 13:00**). Link: 
- [ ] Provide feedback to at least one Assignment 2 from another participant
- [ ] Respond to the GitHub discussion on '[licenses](https://github.com/EstherPlomp/TNW-RDM-101/discussions/34)'
- [ ] Respond to the GitHub discussion on '[folder structure](https://github.com/EstherPlomp/TNW-RDM-101/discussions/36)'
- [ ] Respond to **this** GitHub issue with your readme file
- [ ] Provide feedback to at least one readme file from another participant
- [ ] [Assignment 3](https://estherplomp.github.io/TNW-RDM-101/11-Assignment-Data-Flow-Map-2.html): Data Flow Map 2 (share a link in this issue before **14 March 13:00**). Link:  
- [ ] Provide feedback to at least one Assignment 3 from another participant
- [ ] [Assignment 4](https://estherplomp.github.io/TNW-RDM-101/13-Assignment-DMP.html): Data Management Plan (before **Class 2**)
- [ ] Respond to the GitHub discussion on '[Data Management Plans](https://github.com/EstherPlomp/TNW-RDM-101/discussions/35)' if you have any questions (optional)
- [ ] [Assignment 5](https://estherplomp.github.io/TNW-RDM-101/14-Assignment-Data-Flow-Map-3.html): Data Flow Map 3: submit your slide (before **Class 2**)

**Comment by EstherPlomp** [comment]
Hi @moengels! Thanks for starting the course and handing in assignment 1! I don't see an answer to the question what you would like to learn during the course? Could you please still add something here? Thanks!

**Comment by moengels** [comment]
> Hi Moritz! Your data flow map in assignment 2 looks clear to me! I think you already have a nice idea of which data you will use and how to handle the storage of it. Did you already think about the memory that is needed to store your data?

Hi @JRdeJong, thanks for the feedback. I had to up the limit of what I will acquire. It will boil down to tens of TBs of data in the end in particular after processing. Will probably need Project data folder and a lot of harddrives to back up.

**Comment by EstherPlomp** [comment]
Hi @moengels! Thanks for handing in assignment 2 and for adding what you would like to learn during the course. The course materials don't particularly hit this spot very well, but hopefully you'll have some time to think about this during the course. I'm also going over some resources/materials in the meantime to see if there's anything relevant to share with you: to be continued :) 

I think your Assignment 2 is very extensive/complete, other than indeed the question that @JRdeJong already asked about the size of the data! It looks like you got that covered with the project drive (and indeed a lot of harddrives - extra backups never hurt!) I like how you're also viewing Zenodo as a back up of the data - which it indeed is, next to being an option to publicly share data the data archive will indeed take care of the long term preservation of the data. That saves us a lot of time/head aches in the long run!

**Comment by EstherPlomp** [comment]
I think for your question on metadata for wetlab/imaging data you may want to check out this publication (if you haven't already!): https://doi.org/10.1038/s41592-021-01166-8 and the ISA tools by ELIXIR: https://isa-tools.org/

**Comment by moengels** [comment]
Thank you for the suggestions. 
Besides that, find the Readme file for my folder hosting the training data for a multiplane segmentation model [here](https://surfdrive.surf.nl/files/index.php/s/G9YlnLL6LqBYc0f)

**Comment by EstherPlomp** [comment]
Thanks for sharing Assignment 3 @moengels! It looks very clear - well done! I only have a couple of suggestions: 

Documentation:
Just a note that an alternative to obsidian may be [logseq](https://logseq.com/) - I think they're both comparable so not sure if there's a better option :)

Data publication:
- EMPIAR / Zenodo are great solutions to share data/code 👍. Just a note that if you share software using only GitHub you're not following the TU Delft Research Software policy: according to this policy a snapshot/version of your software should also be shared via a data repository such as Zenodo. This is because GitHub does not guarantee long term preservation of the code (Microsoft could pull out the plug anytime) and GitHub doesn't assign DOIs to the software. See [here for some more information to link your GitHub repositories with Zenodo](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content), which should allow you to automatically upload a snapshot of your code whenever you release a new version on GitHub! 

I'll get back to your READme file later!

**Comment by moengels** [comment]
> Thanks for sharing Assignment 3 @moengels! It looks very clear - well done! I only have a couple of suggestions:
> 
> Documentation: Just a note that an alternative to obsidian may be [logseq](https://logseq.com/) - I think they're both comparable so not sure if there's a better option :)
> 
> Data publication:
> 
> * EMPIAR / Zenodo are great solutions to share data/code 👍. Just a note that if you share software using only GitHub you're not following the TU Delft Research Software policy: according to this policy a snapshot/version of your software should also be shared via a data repository such as Zenodo. This is because GitHub does not guarantee long term preservation of the code (Microsoft could pull out the plug anytime) and GitHub doesn't assign DOIs to the software. See [here for some more information to link your GitHub repositories with Zenodo](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content), which should allow you to automatically upload a snapshot of your code whenever you release a new version on GitHub!
> 
> I'll get back to your READme file later!

Wasn't aware of the Github longterm-preservation issue, will check the link to zenodo for publication code, thanks!

**Comment by EstherPlomp** [comment]
Lots of people using GitHub are not aware of that actually!

regarding your READme: this looks fantastic, well done! 
- a minor point would be to keep in mind when you refer to papers for methodological information. If they are open access it is not so problematic, but if they are behind a paywall not everyone will be able to access the information. Especially in that last case it can be helpful to copy-paste some information from the paper to your readme file. 
- And a question: what does the 'willy' mean in the location information? 
- Great to see that you already have an ORCID!

**Comment by JRdeJong** [comment]
Hi! I wanted to say your READme looks very clean! I especially like how elaborate you are about the folder structure, their relation, but also on the ways of processing the data.

**Comment by EstherPlomp** [comment]
Hi @moengels! The only thing that I think is missing to complete the course is feedback on an assignment 3 and a readme file from another course participant - I could not find your feedback, please let me know if I'm overlooking something! Thanks!

**Comment by moengels** [comment]
> Hi @moengels! The only thing that I think is missing to complete the course is feedback on an assignment 3 and a readme file from another course participant - I could not find your feedback, please let me know if I'm overlooking something! Thanks!

Hi @EstherPlomp, nothing overlooked, is done now.
